tests/cli/test_cli.py

```diff
@@ -56,51 +56,23 @@ def test_cli_clip_help():
     r = CliRunner().invoke(hydromt_cli, ["clip", "--help"])
     assert r.exit_code == 0
     assert r.output.startswith(
-        "Usage: main clip [OPTIONS] MODEL MODEL_ROOT MODEL_DESTINATION REGION"
+        "Usage: main clip [OPTIONS] MODEL MODEL_ROOT MODEL_DESTINATION"
     )
 
 
-@pytest.mark.skip(reason="GridModel has been removed")
 def test_cli_build_grid_model(tmpdir):
     root = str(tmpdir.join("grid_model_region"))
     cmd = [
         "build",
         "grid_model",
         root,
-        "-r",
-        "{'bbox': [12.05,45.30,12.85,45.65]}",
         "--opt",
         "setup_grid.res=0.05",
         "-vv",
     ]
     _ = CliRunner().invoke(hydromt_cli, cmd)
 
 
-@pytest.mark.skip(
```

Comment:
Why are the override tests skipped?

Reply:
I'd have to write another yaml file for that since we don't have a way to specify model components from the CLI. I can do that if you like

Reply:
Does the same `--fo` flag exist now?

Reply:
yes